16_00_cct.Rmd

```diff
@@ -108,7 +108,7 @@ if ( knitr::is_latex_output() ) {
 
 ### Metadata
 
-A major initiative to facilitate data use is the development and implementation of structured +metadata_standards| to be applied to all CCT data. CCT's Data Science team together with the Information and Knowledge Management department have developed standards comprising a minimal required set of [metadata elements](http://osf.io/2a7ev) along with additional, recommended elements. The minimal set of elements is similar to the +Dublin_Core| but with unique considerations for administrative data. The intent of the required minimal set is to minimize the burden on data stewards and to maintain quality metadata. Core fields include a summary of the data set, update frequency, data access rights, reasons for restriction, data format, spatial coverage, temporal coverage, and information about the source and stewardship of the data set.
+A major initiative to facilitate data use is the development and implementation of structured +metadata_standards| to be applied to all CCT data. CCT's Data Science team together with the Information and Knowledge Management department have developed standards comprising a minimal required set of [metadata elements](http://osf.io/2a7ev)[^cctaccessdate] along with additional, recommended elements. The minimal set of elements is similar to the +Dublin_Core| but with unique considerations for administrative data. The intent of the required minimal set is to minimize the burden on data stewards and to maintain quality metadata. Core fields include a summary of the data set, update frequency, data access rights, reasons for restriction, data format, spatial coverage, temporal coverage, and information about the source and stewardship of the data set.
```

Comment:
The OSF repo should be cited in the bibliography, and text modified to "required set of metadata elements (Cole et al, 2020)" (where the Cole et al stems from the [@Cole2020] citation.

Reply:
Likewise with the other OSF links, this is appendix C